### PR TITLE
Update static templates CPLP-1994

### PR DESCRIPTION
**Pull request by manojava-gk**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,14 @@ New features, fixed bugs, known defects and other noteworthy changes to each rel
 
 
 ### Unreleased
+* Dailog Header
+   * Update Dailog header font size to 24px
```
Comment (jjeroch) [inline CHANGELOG.md:8]: @manojava-gk thats already released in the released version below if I am not wrong. Please have a look into ## 1.0.0-RC2

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,14 @@ New features, fixed bugs, known defects and other noteworthy changes to each rel
...
+* Bugfix - Change Email Pattern Regex to fix Hostspot
```
Comment (jjeroch) [inline CHANGELOG.md:9]: same as above

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,14 @@ New features, fixed bugs, known defects and other noteworthy changes to each rel
...
+* LeadImagePicture
+   * Remove asset repo dependency. New api to get image
```
Comment (jjeroch) [inline CHANGELOG.md:11]: same as above

**Review by jjeroch — CHANGES_REQUESTED** [review]

**Comment by jjeroch** [comment]
content wise it looks good - exactly what I was looking for.
It is off course a bit awkward that we locate now the full template inside the locales; but I like the solution much more then having it placed inside the page code. Also it provides the chance to have multi language even for the pictures - which is very good

File: cx-portal-shared-components/src/components/basic/StaticTemplate/Cards/TextCenterAlignedBody2.tsx [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+          {provider.subTitle2}
+        </Typography>
+        <Typography
+          sx={{
+            padding: '30px 0px',
+          }}
+          variant="body2"
+        >
+          {provider.subTitle3}
+        </Typography>
```
Comment (oyo) [inline cx-portal-shared-components/src/components/basic/StaticTemplate/Cards/TextCenterAlignedBody2.tsx:63]: almost same block repeated three times
please change to:
`[provider.subTitle1, provider.subTitle2, provider.subTitle3].map(`
`  (title: string) => <Typography etc>{title}</Topography>`
`)`

**Review by oyo — CHANGES_REQUESTED** [review]

File: cx-portal/src/components/pages/CompanyRoles/index.tsx [hunk 5]
```diff
@@ -20,57 +20,59 @@
...
 import StageSection from 'components/shared/templates/StageSection'
 import { StageSubNavigation } from 'components/shared/templates/StageSubNavigation'
-import { useTranslation } from 'react-i18next'
 import { StaticTemplate } from 'cx-portal-shared-components'
+import CommonService from 'services/CommonService'
 
 export default function CompanyRoles() {
-  const { t } = useTranslation()
-  const [messageContent, setMessageContent] = useState<any>({})
+  const [companyRoles, setCompanyRoles] = useState<any>()
```
Comment (nidhigarg-bmw) [inline cx-portal/src/components/pages/CompanyRoles/index.tsx:29]: fix this datatype

File: cx-portal/src/components/pages/UseCase/index.tsx [hunk 6]
```diff
@@ -19,47 +19,51 @@
...
-      navigation: 'data-id',
-    },
-    {
-      index: 3,
-      title: t('navigation.useCaseNavigation.link3Label'),
-      navigation: 'business-id',
-    },
-  ]
+  useEffect(() => {
+    CommonService.getUseCases((data: any) => {
```
Comment (nidhigarg-bmw) [inline cx-portal/src/components/pages/UseCase/index.tsx:33]: any datatype

File: cx-portal/src/components/pages/CompanyRoles/index.tsx (hunk 5 above)
Comment (manojava-gk) [reply cx-portal/src/components/pages/CompanyRoles/index.tsx:29]: it has to be any :)

File: cx-portal/src/components/pages/UseCase/index.tsx (hunk 6 above)
Comment (manojava-gk) [reply cx-portal/src/components/pages/UseCase/index.tsx:33]: it has to be any :)

**Review by oyo — APPROVED** [review]
lgtm now

**Review by jjeroch — APPROVED** [review]

**Pull request merged**